spring-boot-nashorn/src/main/java/com/baeldung/nashorn/controller/MyWebController.java

```diff
@@ -0,0 +1,29 @@
+package com.baeldung.nashorn.controller;
+
+import java.io.FileReader;
+import java.util.Map;
+
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class MyWebController {
+
+    @RequestMapping("/")
+    public String index(Map<String, Object> model) throws Exception {
+
+        ScriptEngine nashorn = new ScriptEngineManager().getEngineByName("nashorn");
+
+        nashorn.eval(new FileReader("./src/main/webapp/js/react.js"));
```

Comment:
I don't think these src/main paths will work once the app is packaged and deployed